qa/ccs-rolling-upgrade-remote-cluster/build.gradle

```diff
@@ -29,6 +29,7 @@ BuildParams.bwcVersions.withWireCompatible { bwcVersion, baseName ->
     versions = [bwcVersion.toString(), project.version]
     setting 'cluster.remote.node.attr', 'gateway'
     setting 'xpack.security.enabled', 'false'
+    setting 'logger.org.elasticsearch.env.NodeEnvironment', 'TRACE'
```

Comment:
Do we want to keep this in here?

Reply:
No, i've just been adding these for debugging the windows bwc test failure.